### ISSUE TITLE
Improvements to End of Rains/Season dialog layout

### ISSUE THREAD

**Issue by dannyparsons** [africanmathsinitiative/R-Instat#5196]
I'm sure this is in another issue but I can't find it easily and good to have it separate here.
Once my pull request #5195 on the calculation is merged the dialog can also be improved:

1. Checkboxes for End of Rains/Season change to radio buttons at the top of the dialog
2. There is then much more space on the dialog for the controls as the End of Rains controls can be on top of the End of Season controls now that they won't display together. In particular the text boxes for the new column names are very small and could be a lot bigger now. Consider the size and spacing of other controls.
3. More descriptive labels. For End of Rains currently it just says "Amount" and "Over day(s)". I think the dialogs should be more descriptive and give you a good idea of what the calculation might do just from reading it. 
So this could change to "End of rains when > amount:" for example. It would be good for someone to look critically at all the labels and consider if they are clear. This is good for someone who doesn't know the climatic calculations well. If you can't work out what the general idea of what the calculation will do, then its probably not clear enough.

@africanmathsinitiative/developers this would be a good task for someone interested in getting involved in the climatic menu in general. Comment if you are interested - it could be more than one person.

**Comment by Wycklife** [comment]
@africanmathsinitiative/developers I would like to take over on this issue.

**Comment by Ivanluv** [comment]
@africanmathsinitiative/developers  am helping out on this

**Comment by dannyparsons** [comment]
Thanks I have assigned you both. I'll let you know when it can be taken over.

**Comment by dannyparsons** [comment]
My pull request is merged but it's only half complete. I need to still update the code for the End of Season part so its not quite ready for this task yet.

**Comment by maxwellfundi** [comment]
@dannyparsons  a quick reminder on this, please let @Ivanluv  and @Wycklife know when they can take over this job.

**Comment by dannyparsons** [comment]
The end of season code has been updated by https://github.com/africanmathsinitiative/R-Instat/pull/5281.
So this task can now start. Note that there is a lot of code in the dialog so I have tried to organise it into section so please keep the code in the same structure.
